From c1f8a9e819201b59cacc047715b30dd2b4a1df30 Mon Sep 17 00:00:00 2001
From: Yusuke Suzuki <ysuzuki@apple.com>
Date: Mon, 22 Jan 2024 14:17:26 -0800
Subject: [PATCH] Unreviewed, build fix for CLoop
 https://bugs.webkit.org/show_bug.cgi?id=267881 rdar://121391447

* Source/JavaScriptCore/llint/LowLevelInterpreter.cpp:
(JSC::CLoop::execute):

Canonical link: https://commits.webkit.org/273318@main
---
 Source/JavaScriptCore/llint/LowLevelInterpreter.cpp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp b/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp
index 769b582e754b9..75cecbbda6ec2 100644
--- a/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp
+++ b/Source/JavaScriptCore/llint/LowLevelInterpreter.cpp
@@ -331,6 +331,14 @@ JSValue CLoop::execute(OpcodeID entryOpcodeID, void* executableAddress, VM* vm,
     CLoopRegister metadataTable;
     CLoopDoubleRegister d0, d1;
 
+    UNUSED_VARIABLE(t0);
+    UNUSED_VARIABLE(t1);
+    UNUSED_VARIABLE(t2);
+    UNUSED_VARIABLE(t3);
+    UNUSED_VARIABLE(t5);
+    UNUSED_VARIABLE(t6);
+    UNUSED_VARIABLE(t7);
+
     struct StackPointerScope {
         StackPointerScope(CLoopStack& stack)
             : m_stack(stack)
